src/api/index.ts

```diff
@@ -16,5 +16,5 @@ export const queryClient = new QueryClient({
  * TODO: CHANGE BASE URL.
  */
 export const http: AxiosInstance = axios.create({
-  baseURL: 'https://vndbre-proxy.azurewebsites.net/api/v1/',
+  baseURL: import.meta.env.VITE_VNDBRE_PROXY_BASEURL as string,
```

Comment:
There is a better way to add typings https://vitejs.dev/guide/env-and-mode.html#intellisense-for-typescript